src/CMakeLists.txt

```diff
@@ -295,6 +295,9 @@ elseif(ALLOCATOR STREQUAL "jemalloc")
   set(TCMALLOC_srcs perfglue/disabled_heap_profiler.cc)
 elseif(ALLOCATOR STREQUAL "libc")
   set(TCMALLOC_srcs perfglue/disabled_heap_profiler.cc)
+else()
+  set(TCMALLOC_srcs perfglue/disabled_heap_profiler.cc)
+  message(FATAL_ERROR "Possibly incorrect allocator selected:" ${ALLOCATOR})
```

Comment:
nit, add a space after `:`. BTW, could put the variable right into the string to print, also could use a stronger tone here. like
```cmake
message(FATAL_ERROR "Unsupported allocator selected: ${ALLOCATOR}")
```